Modules/HR/Http/Controllers/Universities/ReportController.php

```diff
@@ -16,4 +17,33 @@ public function index()
 
         return view('hr::universities.reports');
     }
+
+    public function show(University $report)
```

Comment:
Why are calling a university object a report?
```suggestion
    public function show(University $university)
```